mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/migration/MailboxPathV3Migration.java

```diff
@@ -106,7 +107,7 @@ public MailboxPathV3Migration(CassandraMailboxPathV2DAO daoV2, CassandraMailboxP
     @Override
     public void apply() {
         daoV2.listAll()
-            .flatMap(this::migrate)
+            .flatMap(this::migrate, CONCURRENCY)
```

Comment:
In order to comply this crashed partial migration we need to:

`.then(daoV2.delete(idAndPath.getPath())))`